src/components/CommentInput.jsx

```diff
@@ -1,53 +1,86 @@
-// import { useEffect, useState } from "react";
-// import { getCommentsByArticleID, postCommentByArticleID } from "../api/api";
+import { useEffect, useState } from "react";
+import { getCommentsByArticleID, postCommentByArticleID } from "../api/api";
 
-// export const CommentInput = (params) => {
-//   const [comment, setComment] = useState("");
-//   const [loading, setLoading] = useState(true);
-//   const [commentSubmitted, setCommentSubmitted] = useState(false);
+export const CommentInput = (params) => {
+  const [comment, setComment] = useState("");
+  const [loading, setLoading] = useState(false);
+  const [commentSubmitted, setCommentSubmitted] = useState(false);
+  const [error, setError] = useState(false)
+  const [commentWarning, setCommentWarning] = useState(false)
+  const [commentEmpty, setCommentEmpty] = useState(false)
 
-//   const { id, onCommentPosted} = params;
+  const { id, onCommentPosted} = params;
 
-//   useEffect(() => {
-//   }, [commentSubmitted]);
+  useEffect(() => {
```

Comment:
im not sure what this is doing

Reply:
oops was left over from some previous code i was testing!! removed now :)